src/components/common/table.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback, useEffect, useState, ReactElement } from 'react';
+import styled from 'styled-components';
+
+interface TableProps<T = any> {
+	headers: string[];
+	data: T[];
+	loader?: ReactElement;
+	loading?: boolean;
+	noData?: ReactElement;
+	initialSort?: { column: number; direction: 'asc' | 'desc' };
+}
+
+const TableTest: React.FC<TableProps> = ({ headers, data, initialSort, loader, loading, noData }) => {
```

Comment:
What is TableTest?

Reply:
TableTest is the component I made for the redesigned table component before I was told to just restyle the table on Assets Page. That's a bad name, sorry I forgot about that, I just gave it that name so it wouldn't conflict with the existing name. Should I delete it or keep it in case we need it in the future?

Reply:
You can delete it